joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/dataflow/DataflowTest.scala

```diff
@@ -678,14 +678,12 @@ class DataflowTest extends DataFlowCodeToCpgSuite {
 
     "literal to captured closure" in {
       val literalSource = cpg.literal.codeExact("\"https://test-api-service.com\"").l
-      literalSource.size shouldBe 1
```

Comment:
I think we should keep the `literalSource.size` check, since it helps us debug the test if it fails

Reply:
Ok, will update this